src/index.css

```diff
@@ -1 +1,9 @@
+
+
+
+body {
+  background-color: white;
+}
+
 @import-normalize;
```

Comment:
We might want to leave this normalize at the top of the sheet so it gets applied to everything first